node_api/src/logic/IMQtoSQL.ts

```diff
@@ -0,0 +1,371 @@
+import { Match, Query, Property, Assignable, OrderLimit } from "@im-library/interfaces/AutoGen";
+import { SqlQuery } from "@/model/sql/SqlQuery";
+
+export class IMQtoSQL {
```

Comment:
class doesn't have any fields or values that are needed within the context of an object - could convert to a list of functions (doesn't need to be an object)